website/docs/add-secure-apps/providers/ssf/index.md

```diff
@@ -1,30 +1,45 @@
+An SSF provider allows applications to subscribe to _security signals_, translating them into Security Event Tokens (<abbr>SET</abbr>) that are captured by the IdP. In this arrangement, authentik acts as an mediator from an external event _transmitter_, ultimately with a specific application acting as the _receiver_ of the events.
 
-Events in authentik that are tracked via SSF include when an MFA device is added or removed, logouts, sessions being revoked by Admin or user clicking logout, or credentials changed.
+### Transmitter and receiver
+
+The role of the transmitter is to send signals to the receiver, and the receiver is the application that listens for the signals. Each system receives and sends signals, but the signals are different.
+
+The identity provider is the transmitter of _identity signals_ through authentik to the application,
+and the application is the receiver of _identity signals_, such as user information, groups, and roles.
+
+An SSF provider is the transmitter of _security signals_ through authentik to the application,
+and the application is the receiver of _security signals_, such as device information, user events, and security events.
+
+This shared signals approach allows for more dynamic security decisions than traditional static policies, as access determinations can adapt based on both identity context and device posture in near real-time.
+
+:::info Roles vary by integration
+
+The arrangement of transmitter and receiver will vary depending on the integration. And in some cases, such as IdP-to-IdP communication, the transmitter and receiver can be different systems entirely.
+
+:::
 
 ## Example use cases
 
 A common use case for SSF is when an Admin wants to know if a user logs out of authentik, so that the user is then also automaticlaly logged out of all other work-focused applications.
 
-Another example use case is when an application uses SSF to subscribe to authorization events because the application needs to know if a user changed their password in authentik. If a user did change their password, then the application receives a POST request to write the fact that the password was changed.
-
-## About using SSF in authentik
+Another example use case is when an application uses SSF to subscribe to authorization events because the application needs to know if a user changed their password in authentik. If a user changes their password in authentik, an event will be generated and sent to the application, which can then take action based on the content of the event.
 
-Let's look at a few details about using SSF in authentik.
+## SSF in authentik
```

Comment:
We do not need this header. There is already a header, called "About using SSF in authentik". This is sufficient. And why do you want to remove line 25?